_episodes/03-computing-pi.md

```diff
@@ -200,6 +222,7 @@ We now build a queue/worker model. This is the basis of multi-threading applicat
 this point creating a parallel program is quite involved. After we've done this, we'll see ways to
 do the same in Dask without mucking about with threads directly.
 
+FIXME: Define the `sum_primes` function
```

Comment:
good catch! In this case it should be any of the `calc_pi` variants. I copy pasted this code from a different tutorial ;)

Reply:
Ok, I am now using the `calc_pi` function. Kind of curious what would happen if you used `calc_pi_dask` (I guess that threading would not actually make it faster).